### PR TITLE
More dummy text

### PR DESCRIPTION
**Pull request by BachirC**
ok

### CODE REVIEW

File: sample.txt [hunk 1]
```diff
@@ -1 +1 @@
-Some text
+Some text updated
```
Comment (Mszniber) [inline sample.txt:1]: C'est cool

**Comment by Mszniber** [comment]
coucou

File: sample.txt (hunk 1 above)
Comment (Mszniber) [inline sample.txt:1]: bien

Comment (Mszniber) [inline sample.txt:1]: tbarkellah 3lik

Comment (Mszniber) [inline sample.txt:1]: GG

**Comment by BachirC** [comment]
noyce

**Comment by BachirC** [comment]
@BachirC

**Comment by BachirC** [comment]
@BachirC

**Comment by BachirC** [comment]
@BachirC Wtf are you doing

**Comment by BachirC** [comment]
@BachirC I think this feature sucks

**Comment by BachirC** [comment]
I think what @BachirC is saying is kinda clever and stupid at the same time. What a genius.

**Pull request closed without merging**